.github/workflows/ci-production.yml

```
@@ -22,6 +22,7 @@ jobs:
       env:
           NWB_CONVERSION_INSTALL_MODE: production
       run: |
+        pip install -r requirements-dev.txt
```

```suggestion
      run: |
        pip install -r requirements.txt
```
The ci-production tests are the ones that test the 'laziness' of each of the interfaces that have additional requirements; the `requirements-dev.txt` installs every possible dependency for every possible interface.